.github/workflows/ci_job.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           http://localhost:8080/realms/freva/.well-known/openid-configuration
       - name: Starting the local server
         run: |
-          python run_server.py -c api_config.toml --debug --dev -p 7777
+          python run_server.py --config api_config.toml --debug --dev -p 7777
```

Comment:
```bash
python run_server.py --config freva-rest/api_config.toml --debug --dev -p 7777
```

Reply:
See my comment above. Would you be more happy if this file would be saved under `dev-env`? The actual devault api_config.toml file is already shipped with the pip package in the src folder.